### DISCUSSION TITLE
Feature request: toggle boolean value using increment/decrement shortcuts

### DISCUSSION THREAD

**Discussion by aral** [helix-editor/helix#10198] [General]
#### Describe your feature request

If the token under the cursor(s) is a boolean `true` or `false`, toggle its value when the increment or decrement commands are issued.

**Comment by A-Walrus** [comment]
This should probably support both capitalized `True / False` (like in Python), lowercase `true / false` (like in Rust), and uppercase `TRUE / FALSE` (like in COBOL)

**Comment by isimons** [comment]
was this feature implemented somehow?

**Comment by stibinator** [comment]
The ability to in/de-crement through a user supplied list would be great. For example when working with graphics you could toggle [width, height], [x, y], or step through [Red, Green, Blue, Alpha] etc. By making it user-configurable it would be useful for a whole range of use cases.
